src/main/java/com/stripe/net/HttpClient.java

```diff
@@ -63,23 +74,40 @@ public StripeResponse requestWithTelemetry(StripeRequest request) throws StripeE
   }
 
   /**
-   * Sends the given request to Stripe's API, retrying the request in cases of intermittent
-   * problems.
+   * Sends the given request to Stripe's API, handling telemetry if not disabled.
    *
    * @param request the request
    * @return the response
    * @throws StripeException If the request fails for any reason
    */
-  public StripeResponse requestWithRetries(StripeRequest request) throws StripeException {
+  public StripeResponse requestWithTelemetry(StripeRequest request) throws StripeException {
+    return sendWithTelemetry(request, (r) -> this.request(r));
```

Comment:
Nit: Can use method references here throughout if you want to save some typing:

```
(r) -> this.request(r)
```

can become

```
this::request
```